k-simplex2vec_Introduction.ipynb

```diff
@@ -68,8 +68,8 @@
     "for edge in SBM.edges():\n",
     "    st.insert(list(edge))\n",
     "st.expansion(3) ## Fills in the clique complex up to dimension 3\n",
-    "simps_2= len(st.get_skeleton(2))-len(st.get_skeleton(1))\n",
-    "simps_3 = len(st.get_skeleton(3))-len(st.get_skeleton(3))\n",
+    "simps_2= len(list(st.get_skeleton(2)))-len(list(st.get_skeleton(1)))\n",
```

Comment:
71: here st.get_skeleton() returns a generaotr, thus, we have to use list() before len() to find the length of the skeleton

72: I assume that you want to compute the number of the 3-simplices, so the len(st.get_skeleton(2)) should be minused instead of len(st.get_skeleton(3))